skelebot/systems/execution/executor.py

```diff
@@ -67,7 +67,8 @@ def executeJob(config, args, job):
     else:
         if (not args.skip_build_global):
             buildDocker(config)
-        status = runDocker(config, command, job.mode, config.ports, job.mappings, job.name)
+        ports = config.ports + job.ports
```

Comment:
Maybe remove duplicates here, in case users have the same port mapping in multiple places

Reply:
Updated